mimic/test/dummy.py

```diff
@@ -91,3 +91,38 @@ def resource(self):
         """
         example_resource = ExampleResource(self._response)
         return example_resource
+
+
+@implementer(IExternalAPIMock, IPlugin)
+class ExampleExternalAPI(object):
+    """
+    Example external API the return nothing
+    """
+
+    def __init__(self, name=u"example",
+                 regions_and_versions=[
+                     ('ORD', 'v1',
```

Comment:
Could we pick a region other than ORD for testing here so it's easier for readers of the tests to recognize it immediately as distinct from other places where "ORD" is a placeholder? Maybe just `"EXTERNAL"`?